src/Service/Messenger.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Service;
+
+
+use App\Model\Message;
+
+class Messenger
+{
+    /**
+     * @var SenderInterface[]
+     */
+    protected $senders = [];
+
+    /**
+     * Messenger constructor.
+     * @param SenderInterface[] $senders
+     */
+    public function __construct(array $senders = [])
+    {
+        $this->senders = $senders;
+    }
+
+    /**
+     * {@inheritDoc}
+     *
+     */
+    public function send(Message $message)
```

Comment:
No return type defined;
Should this method really return something (e.g. whether sending was successful) or is `return` here only used for short-circuiting the code flow once a supporting sender is found and the return type should be `void`?